src/Microsoft.IdentityModel.Logging.Abstractions/Microsoft.IdentityModel.Logging.Abstractions.csproj

```diff
@@ -0,0 +1,29 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="..\..\build\common.props" />
+
+  <PropertyGroup>
+    <AssemblyName>Microsoft.IdentityModel.Logging.Abstractions</AssemblyName>
```

Comment:
>Logging [](http://example.com/codeflow?start=42&length=7)

Can this be Microsoft.IdentityModel.Observability.Abstractions?

That way it will be a more natural home for other telemetry stuff as well

Reply:
Talked offline. I believe there was another suggestion of MS.ID.Abstractions which could serve multiple purposes.

Reply:
@TimHannMSFT did you mean M.IM.Abstractions?

Reply:
Yup, he meant M.IM.Abstractions.